client/src/containers/Manifold.js

```diff
@@ -183,6 +183,7 @@ class ManifoldContainer extends PureComponent {
     );
 
     return (
+      /* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
This seems presentational. The onClick handler here is used to hide open navigation when the user clicks outside of them. It's not necessary for a screen reader to know about this.

Reply:
Agreed.  And in fact after additional research and testing using the `role="presentation"` will work quite well in these kinds of situations as it only unsets the role for the target element (only has an effect on direct children who depend on the parent for their role -- i.e. setting a list to presentation would cause list items to turn into text elements in screen readers).  I will also use this `role="presentation"` strategy to address those problem spots called out in case 1 in the PR description.